roofit/roofitcore/inc/RooConstraintSum.h

```diff
@@ -44,6 +44,7 @@ class RooConstraintSum : public RooAbsReal {
         RooArgSet const* globalObservables,
         const char* globalObservablesTag,
         bool takeGlobalObservablesFromData,
+        bool cloneConstraints,
```

Comment:
Isn't this a backward incompatible change? This will misinterpret user code without that new `bool` parameter:
```
...createConstraintTerm(..., "globalObservablesTag", true /*takeGlobalObservablesFromData*/, worksp);
```
will now instead be interpreted as `cloneConstraints = (bool) worksp` with `workspace = nullptr`! *If* this really needs to happen it should be called out with big letters in the release notes! But I'd very much prefer not to build this trap, for instance by
- moving the new parameter to the end
- adding an overload (say `createConstraintTermWithClonedConstraints`)

Reply:
Fortunately it's not a backwards incompatible change, because the helper function `createConstraintTerm` was only introduced in this release cycle :)